docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```petab1
+c1 = Dict(:S => 1.0)
+c2 = Dict(:S => 0.5)
+nothing # hide
+```
+Similarly as for observables, we then gather the conditions in another `Dict`, giving each an id tag:
+```petab1
+simulation_conditions = Dict("c1" => c1, "c2" => c2)
+nothing # hide
+```
+Again (like for observables), each measurement in the measurements `DataFrame` needs to be associated with a simulation condition id tag (describing which condition that measurements were taken from). Parameters, just like initial conditions, may vary between different conditions. If an initial condition (or parameter) occurs in one condition, it must occur in all of them.
+
+Here follows a complete version of our basic example, but with measurements both for *S(0)=1.0* and *S(0)=0.5*.
+```petab3
+using Catalyst, PEtab
+
+rn = @reaction_network begin
+    kB, S + E --> SE
+    kD, SE --> S + E
+    kP, SE --> P + E
+end
+
+u0 = [:E => 1.0, :SE => 0.0, :P => 0.0]
+p_true = [:kB => 1.0, :kD => 0.1, :kP => 0.5]
+
+# Simulate data.
+using OrdinaryDiffEq, Random
+d1, t1 = let 
+    oprob = ODEProblem(rn, [:S=>1.0; u0], (0.0, 10.0), p_true)
+    sol = solve(oprob, Tsit5(); saveat=0.1)
+    ((0.8 .+ 0.4*rand(10)) .* sol[:P][10:10:end], sol.t[10:10:end])
+end
+d2, t2 = let 
+    oprob = ODEProblem(rn, [:S=>0.5; u0], (0.0, 10.0), p_true)
+    sol = solve(oprob, Tsit5(); saveat=0.1)
+    ((0.8 .+ 0.4*rand(10)) .* sol[:P][10:10:end], sol.t[10:10:end])
+end
+
+@unpack P = rn
+obs_P = PEtabObservable(P, 0.5)
+observables = Dict("obs_P" => obs_P)
+
+par_kB = PEtabParameter(:kB)
+par_kD = PEtabParameter(:kD)
+par_kP = PEtabParameter(:kP)
+params = [par_kB, par_kD, par_kP]
+
+c1 = Dict(:kB => 1.0)
+c2 = Dict(:kB => 0.5)
+simulation_conditions = Dict("c1" => c1, "c2" => c2)
+
+using DataFrames
+m1 = DataFrame(simulation_id="c1", obs_id="obs_P", time=t1, measurement=d1)
+m2 = DataFrame(simulation_id="c2", obs_id="obs_P", time=t2, measurement=d2)
+measurements = vcat(m1,m2)
+
+petab_model = PEtabModel(rn, simulation_conditions, observables, measurements, params; state_map=u0)
+nothing # hide
+```
+Note that the `u0` we pass into `PEtabModel` through the `state_map` argument no longer contains the value of *S* (as it is provided by the conditions). 
+
+### [Varying parameters between different simulation conditions](@id petab_simulation_conditions)
+Sometimes, the parameters that are used vary between the different conditions. Consider our catalysis example, if we had performed the experiment twice, using two different enzymes with different catalytic properties, this could have generated such conditions. The two enzymes could e.g. yield different rates (*kP₁* and *kP₂*) for the `SE --> P + E` reaction, but otherwise be identical. Here, the parameters *kP₁* and *kP₂* are unique to their respective conditions. PEtab.jl provides support for cases such as this, and [its documentation](https://sebapersson.github.io/PEtab.jl/dev/Beer/) provided instructions of how to handle them.
```

Comment:
```suggestion
Sometimes, the parameters that are used vary between the different conditions. Consider our catalysis example, if we had performed the experiment twice, using two different enzymes with different catalytic properties, this could have generated such conditions. The two enzymes could e.g. yield different rates (*kP₁* and *kP₂*) for the `SE --> P + E` reaction, but otherwise be identical. Here, the parameters *kP₁* and *kP₂* are unique to their respective conditions. PEtab.jl provides support for cases such as this, and [its documentation](https://sebapersson.github.io/PEtab.jl/stable/Julia_condition_specific/) provided instructions of how to handle them.
```